src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -642,6 +642,58 @@ function GlobeControls(view, target, radius, options = {}) {
             this.camera.position.copy(cameraTargetOnGlobe.localToWorld(offset));
         }
 
+        // HANDLING CAMERA COLLISION WITH TERRAIN
+        // We check here if the camera is too close to the ground or even under it to avoid collisions
+        const camLocation = this.getCameraLocation();
+        let currentTile;
+
+        function tO(obj) {
+            if (obj.extent) {
+                if (obj.extent.isPointInside(camLocation)) {
+                    currentTile = obj;
+                }
+            }
+        }
+
+        if (view.wgs84TileLayer) {
+            for (const node of view.wgs84TileLayer.level0Nodes) {
+                if (node.extent.isPointInside(camLocation)) {
+                    node.traverse(tO);
+                }
+            }
+        }
+
+        // We first test if camera is very close to the tile bounding box
+        if (currentTile) {
+            const alti = currentTile.OBB().z.max + this.minDistance;
+            const distance = camLocation.altitude() - alti;
+            // If inside, we then need more precision: the exact elevation under camera location
+            if (distance < 0) {
+                const corners = currentTile.OBB()._cPointsWorld(currentTile.OBB()._points());
+                const pNormed = this.camera.position.clone().sub(corners[1]);
+                const AB = corners[0].clone().sub(corners[1]);
+                const AC = corners[2].clone().sub(corners[1]);
+                const pNormedProjectedOnAB = pNormed.clone().projectOnVector(AB);
+                const pNormedProjectedOnAC = pNormed.clone().projectOnVector(AC);
+                let u = pNormedProjectedOnAB.length() / AB.length();
+                let v = pNormedProjectedOnAC.length() / AC.length();
+                const offsetScale = currentTile.material.uniforms.offsetScale_L00.value[0];
+                u = offsetScale.x + u * offsetScale.z;
+                v = offsetScale.y + v * offsetScale.z;
+                if (currentTile.material.textures[0][0].image) {
+                    const w = currentTile.material.textures[0][0].image.width;
+                    const indiceElevation = Math.floor(v * w) * w + Math.floor(u * w);
```

Comment:
In rigor, shouldn't it be 
```js
const indiceElevation = Math.floor(v * h) * w + Math.floor(u * w);
```
with `h = image.height`? (please check this though, I might have mixed up the axis).

In practice, I don't know if we'll have non-squared image, but I also find it easier to understand this way.